docs/BreakingChanges/168 DataContractJsonSerializer incompatible with ES6 (and V8).md

```diff
@@ -0,0 +1,33 @@
+## 168: DataContractJsonSerializer incompatible with ES6 (and V8)
+
+### Scope
+Edge
+
+### Version Introduced
+4.7
+
+### Source Analyzer Status
+Planned
+
+### Change Description
+.NET 4.7 fixed the compatibility issue between the escaped strings for some special control characters generated by DataContractJsonSerializer and ECMAScript V6. 
```

Comment:
Change wording:
".NET 4.7 fixed the compatibility issue between the escaped strings for some special control characters generated by DataContractJsonSerializer and ECMAScript V6."
to
"Starting in .NET 4.7, the compatibility issue between escaped strings for some special control characters generated by DataContractJsonSerializer and ECMAScript V6 has been fixed."